### ISSUE TITLE
Widgettini tondi simpa

### ISSUE THREAD

**Issue by Seiseisei666** [Seiseisei666/Quantum#18]
Sto scrivendo per dei bottoncini tondi con grafica matta, che rimpiazzeranno i bottoni quadrati per quanto riguarda il click destro (o il tap con il dito, quando faremo la conversione per touch =P)

A qualcuno va di disegnarli?

**Comment by axstu** [comment]
posso farlo io appena mi metto in pari con tutte le modifiche che avete
fatto questi giorni
Il 15/set/2015 19:55, "Seiseisei666" <notifications@github.com> ha scritto:

**Comment by kAtoblepO** [comment]
Ho un'idea troppo scifi per non implementarla!
Il click destro apre un menù circolare tipo questo:

http://i.stack.imgur.com/OyXjB.png

**Comment by Seiseisei666** [comment]
mmm indicativamente si può fare, ma bisognrebbe disegnare delle sprite molto diverse (non penso che quel menù si possa gestire ruotando le sprite, credo sia disegnato così e basta)
Oggi sistemo un po' i miei widget comunque, li aveti visti che bellini? :heart_eyes: 
Rimetto pure in sesto i meccanismi di tutte le azioni così che ripristiniamo le funzionalità pre-serata malgioglio

**Comment by kAtoblepO** [comment]
No, l'ultima volta che ho messo le mani in pasta era la sera del mega-merge.
Il menu tondo nella foto è troppo complesso, però si può fare una cosa semplice tipo così (i cerchietti potrebbero partire dal centro - punto in cui clicchi - e poi spostarsi verso le 2 direzioni) :

![image](https://cloud.githubusercontent.com/assets/13572859/9908182/c06a84de-5c92-11e5-87de-c4e95731600d.png)

**Comment by Seiseisei666** [comment]
Dai un'occhiata al main, la mia idea era proprio questa qui dei cerchietti (o almeno molto simile).
Dimmi che ne pensi!

Al momento ogni cerchietto è posizionato autonomamente, per farli muovere insieme in modo intelligente servirebbe una classe di WidgetManager che ne istanzia N e si occupa di gestirli, ma dato che noi non avremo mai bisogno di più di 2 bottoncini (massimo 3 nella versione touch ^__^) non so quanto ne valga la pena!
